Update release strategy (#404)
kyma-project/community: guidelines/releases-guidelines/04-release-communication.md

## Communication channels

> The Kyma team must be notified about the planned dates of both release candidates, as well as the date of the final release. Apart from that, the Release Master must inform about any occurrences regarding the release process. After the release is out, the Release Master sends notification about the new release on the external Kyma community [Slack workspace](http://slack.kyma-project.io/) **release** channel.

Reviewer: ```suggestion
The Release Manager must notify the Kyma team about the planned dates of creating both release candidates, as well as the date of the final release. Apart from that, the Release Master must inform about the progress of the release process. After the release is out, the Release Master sends notification about the new release on the external **release** [Slack](http://slack.kyma-project.io/) channel.
```
I think that now not only the information that the release is out but whole communication about the release takes place on the public **release** Slack channel, isn't it? Also, line 137 says that it is the Release Manager that notifies the Kyma community on the public **release** Slack channel when a release candidate is available. So maybe we could add the Release Manager to this section as well?
